views/site/login.php

```diff
@@ -7,13 +7,16 @@
 use yii\helpers\Html;
 use yii\bootstrap\ActiveForm;
 
-$this->title = 'Login';
+$this->title = Yii::t('app', 'Login');
 $this->params['breadcrumbs'][] = $this->title;
 ?>
 <div class="site-login">
     <h1><?= Html::encode($this->title) ?></h1>
 
-    <p>Please fill out the following fields to login:</p>
+    <p>
+        <?php echo Yii::t('app/messages','Please fill out the following fields to login:')
```

Comment:
Use <?= ?> instead of <?php  echo ... ?> and add the _?>_  on the same line : 
`<?= Yii::t('app/messages', '....') ?>`

Reply:
`<?=` instead of `<?php` ?

Reply:
for the record : `<?=` instead of `<?php echo`